File: app/src/main/java/com/xpf/p2p/widget/MyTextView.java
package com.xpf.p2p.widget;

import android.content.Context;
import android.util.AttributeSet;

import androidx.appcompat.widget.AppCompatTextView;

/**
 * Created by xpf on 2016/11/19 :)
 * Wechat:vancexin
 * Function:自定义TextView跑马灯效果
 */

public class MyTextView extends AppCompatTextView {

    public MyTextView(Context context) {
        this(context, null);
    }

    public MyTextView(Context context, AttributeSet attrs) {
        this(context, attrs, 0);
    }

    public MyTextView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    @Override
    public boolean isFocused() {
        return true;
    }
}

File: app/src/main/java/com/xpf/p2p/widget/MyTextView.kt
package com.xpf.p2p.widget

import android.content.Context
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatTextView

/**
 * Created by xpf on 2016/11/19 :)
 * Wechat:vancexin
 * Function:自定义TextView跑马灯效果
 */
class MyTextView @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyleAttr: Int = 0
) : AppCompatTextView(context, attrs, defStyleAttr) {

    override fun isFocused(): Boolean = true
}
